packages/core/src/types.ts

```diff
@@ -600,7 +600,7 @@ export interface StateNodeConfig<
    */
   states?: StatesConfig<TContext, TEvent, TAction> | undefined;
   /**
-   * The services to invoke upon entering this state node. These services will be stopped upon exiting this state node.
+   * The actors to invoke upon entering this state node. These actors will be stopped upon exiting this state node.
```

Comment:
are those actors to invoke though? 🤔 actors are definitely invoked as the end result, but the `invoke` property holds behaviors/behavior factories